tests/test-runner.cc

```diff
@@ -77,11 +77,23 @@ compartment_error_handler(struct ErrorState *frame, size_t mcause, size_t mtval)
 	return ErrorRecoveryBehaviour::InstallContext;
 }
 
+__attribute__((noinline, weak)) void *pcc_as_sentry()
+{
+	return __builtin_return_address(0);
+}
+
 /**
  * Test suite entry point.  Runs all of the tests that we have defined.
  */
+[[cheriot::interrupt_state(disabled)]]
```

Comment:
This will only apply to the top level test runner, right? All the tests that run in compartments will default to interrupts enabled?

Reply:
Yes, otherwise the test suite would have broken.